vllm/entrypoints/openai/serving_engine.py

```diff
@@ -78,6 +81,7 @@ def __init__(
 
         self.served_model_names = served_model_names
 
+        self.lora_id_counter = AtomicCounter(0)
```

Comment:
@Jeffwan this is not required here. asyncio operations all happen in the same thread. Can change this to be a simple int field.

Reply:
@njhill yeah, that makes sense. Let me file a follow up PR to improve it.